src/main/scala/cromwell/Main.scala

```diff
@@ -48,15 +50,24 @@ object Main extends App {
 
   def run(args: Array[String]): Unit = {
     if (args.length != 2) usageAndExit()
-    val runner = new SingleWorkflowRunner
-    val outputs = runner.run(new File(args(0)), new File(args(1)))
-    outputs match {
-      case Success(o) =>
-        import cromwell.binding.values.WdlValueJsonFormatter._
-        println("Workflow Completed.  Outputs are:")
-        println(o.toJson.prettyPrint)
-        runner.workflowManagerActor ! WorkflowManagerActor.Shutdown
-      case Failure(f) => println(f.printStackTrace())
+
+    try {
+      val wdl = FileUtil.slurp(Paths.get(args(0)))
+      val jsValue = FileUtil.slurp(Paths.get(args(1))).parseJson
+
+      val inputs: binding.WorkflowRawInputs = jsValue match {
+        case JsObject(rawInputs) => rawInputs
+        case _ => throw new RuntimeException("Expecting a JSON object")
+      }
+
+      val workflowManagerSystem = new WorkflowManagerSystem {}
+      val singleWorkflowRunner = SingleWorkflowRunnerActor.props(wdl, inputs, workflowManagerSystem.workflowManagerActor)
+      val actor = workflowManagerSystem.actorSystem.actorOf(singleWorkflowRunner)
+      // And now we just wait for the magic to happen
```

Comment:
Discussed in person. This serves the purpose for the "launched directly from main as the command line single workflow" use case and we have the server for multi-workflow use case. This is inadequate for a world where we have an API based calling, but we're not currently supporting that. If we end up doing so in the future, this should change.